src/lib/find-changed-projects/find-changed-projects.ts

```diff
@@ -0,0 +1,13 @@
+import { isChangeInPath } from "@src/util/git-client";
+
+export const findChangedProjects = (
+  diffTarget: string,
+  rushProjects: RushProject[]
+) => {
+  return rushProjects.reduce<RushProject[]>((changes, project) => {
+    if (isChangeInPath(diffTarget, project.projectFolder)) {
+      changes.push(project)
+    }
+    return changes;
+  }, []);
+};
```

Comment:
We should agree on a team level I think how we write these returns.
`{ return something... }` or `(something...)`

Reply:
Sure write it down to conventions.